jobs/involuntary-dissolutions/flags.json

```diff
@@ -2,6 +2,19 @@
     "flagValues": {
         "enable-involuntary-dissolution": true,
         "disable-dissolution-sftp-bcmail": false,
-        "disable-dissolution-sftp-bclaws": false
+        "disable-dissolution-sftp-bclaws": false,
+        "db-versioning": {
+            "legal-api": true,
+            "emailer": true,
+            "filer": false,
+            "entity-bn": true,
+            "digital-credentials": false,
+            "dissolutions-job": true,
+            "furnishings-job": true,
+            "emailer-reminder-job": false,
+            "update-colin-filings-job": false,
+            "future-effective-job": false,
+            "update-legal-filings-job": false
```

Comment:
`future-effective-job` and `update-legal-filings-job` have been removed from FFs, you can check the current db-versioning FFs in legal-api (https://github.com/bcgov/lear/blob/feature-db-versioning/legal-api/flags.json)

Reply:
Also, please consider removing the `update-colin-filings-job` flag here